lifecycle-operator/controllers/common/evaluation/evaluationhandler.go

```diff
@@ -20,9 +21,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+//go:generate moq -pkg fake -skip-ensure -out ./fake/evaluationhandler_mock.go . EvaluationHandlerInterface:MockEvaluationHandler
+type EvaluationHandlerInterface interface {
```

Comment:
Let's rename this, we should not use `Interface` suffix in the names

Reply:
I will rewrite it to `IEvaluationHandler`